plugin-dev/Source/Sentry/Private/SentrySubsystem.cpp

```diff
@@ -47,13 +48,21 @@ void USentrySubsystem::Deinitialize()
 {
 	DisableAutomaticBreadcrumbs();
 
+	Close();
```

Comment:
Is that an earlier oversight?

Reply:
Yeah, noticed it while disabling the event capturing for Editor - without shutting down the native SDK upon subsystem's deinitialization events were still passing through in the subsequent game sessions even though the corresponding target was disabled